resolwe/flow/managers/celery.py

```diff
@@ -7,25 +7,38 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import logging
 import sys
 
+from resolwe.utils import BraceMessage as __
+
 from ..tasks import celery_run
 from .base import BaseManager
 
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
 try:
     import celery  # pylint: disable=unused-import
 except ImportError:
-    print('Please install Celery using `pip install celery`', file=sys.stderr)
+    print("Please install Celery using `pip install celery`", file=sys.stderr)
     sys.exit(1)
 
 
 class Manager(BaseManager):
-    """Celey-based manager for job execution."""
+    """Celery-based manager for job execution."""
 
-    def run(self, data_id, script, priority='normal', run_sync=False, verbosity=1):
+    def run(self, data_id, dest_dir, argv, priority='normal', run_sync=False, verbosity=1):
```

Comment:
Please, document `run` method's arguments and keyword arguments in its docstring.

Maybe you can refer to `BaseManager`'s  `run` method to avoid repetition?